makefile

```diff
@@ -1,4 +1,5 @@
 test:
+	v run ./fmt.vsh
```

Comment:
it may be better if you separate it into another make target:
```make
fmt:
   v run ./fmt.vsh
```
The benefit will be that users could do `make test` without having their source code overwritten over and over (`make test` usually assumes that tests are non destructive), while it would be still easy to use `make fmt`, or even `make fmt test` if someone wants to run both.

Reply:
Yeah... Will do in the next commit... Thanks @spytheman !